.github/workflows/find-smallest-rust.yml

```diff
@@ -0,0 +1,42 @@
+name: Minimum Supported Rust
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/find-smallest-rust.yml
+      - core/rust/*
+      - vm/rust/*
+      - starknet/rust/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  find-smallest-rust:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Find smallest supported Rust version
+        id: rust-version
+        uses: derrix060/detect-rust-minimum-version@v1
```

Comment:
should we consider moving the action from your personal account to the NethermindEth org? It might help with better control, maintenance, and security in the long run.

Reply:
well, this is actually not nethermind specific, it is just a simple wrapper around cargo-msrv which can be used by any organisation

Reply:
Yep, it’s generic, but still, moving your action to the NethermindEth org could help with more control and maintenance. I remember the core team had a similar case and preferred hosting things internally rather than relying on non-official actions hosted on individual accounts.